src/main/java/net/fabricmc/loom/LoomGradleExtension.java

```diff
@@ -119,6 +119,8 @@ default String getIntermediaryUrl(String minecraftVersion) {
 	// ===================
 	//  Architectury Loom
 	// ===================
+	Project getProject();
```

Comment:
IMO this is wrong, if you have the extension then you should already have the project from elsewhere